src/OneExplorer.ts

```diff
@@ -77,15 +71,15 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
   }
 
   getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
-    if (this.oneTree === undefined) {
-      vscode.window.showInformationMessage('No ONE model or config in empty workspace');
-      return Promise.resolve([]);
-    }
-
     if (element) {
       return Promise.resolve(this.getNode(element.node));
     } else {
-      return Promise.resolve(this.getNode(this.oneTree));
+      if (!this.workspaceRoot) {
```

Comment:
How about locating this function outside this `if-else` phrase?
This change does not change the operation, but it's easier to understand the code.

Note.
If the `this.workspaceRoot` is undefined, element is always undefined.